pkgs/bootandy/dust/registry.yaml

```diff
@@ -4,22 +4,129 @@ packages:
+        format: tar.gz
+        rosetta2: true
+        replacements:
+          amd64: x86_64
+          darwin: apple-darwin
+        overrides:
+          - goos: linux
+            goarch: amd64
+            replacements:
+              linux: unknown-linux-musl
+          - goos: linux
+            goarch: arm64
+            replacements:
+              arm64: aarch64
+              linux: unknown-linux-gnu
+        supported_envs:
+          - linux
+          - darwin
+      - version_constraint: semver("<= 0.3.1")
+        asset: dust-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
+        format: tar.gz
+        rosetta2: true
+        replacements:
+          amd64: x86_64
+          darwin: apple-darwin
+          linux: unknown-linux-gnu
+        supported_envs:
+          - linux/amd64
+          - darwin
+      - version_constraint: Version == "v0.4.0.1"
+        asset: dust-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
+        format: tar.gz
+        rosetta2: true
+        replacements:
+          amd64: x86_64
+          darwin: apple-darwin
+          linux: unknown-linux-gnu
+        supported_envs:
+          - linux/amd64
+          - darwin
+      - version_constraint: semver("<= 0.4.4")
+        asset: dust-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
+        format: tar.gz
+        rosetta2: true
+        replacements:
+          amd64: x86_64
+          darwin: apple-darwin
+          linux: unknown-linux-gnu
+        supported_envs:
+          - linux/amd64
+          - darwin
+      - version_constraint: semver("<= 0.6.2")
+        asset: dust-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
+        format: tar.gz
+        rosetta2: true
+        windows_arm_emulation: true
+        files:
+          - name: dust
+            src: "{{.AssetWithoutExt}}/dust"
+        replacements:
+          amd64: x86_64
+          darwin: apple-darwin
+          windows: pc-windows-msvc
+        overrides:
+          - goos: linux
+            goarch: amd64
+            replacements:
+              linux: unknown-linux-musl
+          - goos: linux
+            goarch: arm64
+            asset: dust-{{.Version}}-{{.Arch}}-{{.OS}}eabihf.{{.Format}}
+            replacements:
+              arm64: arm
+              linux: unknown-linux-gnu
+          - goos: windows
+            format: zip
+      - version_constraint: semver("<= 0.8.0")
+        asset: dust-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}
+        format: tar.gz
+        rosetta2: true
+        windows_arm_emulation: true
+        files:
+          - name: dust
+            src: "{{.AssetWithoutExt}}/dust"
+        replacements:
+          amd64: x86_64
+          darwin: apple-darwin
+          windows: pc-windows-msvc
+        overrides:
+          - goos: linux
+            goarch: amd64
+            replacements:
+              linux: unknown-linux-musl
+          - goos: linux
+            goarch: arm64
+            asset: dust-{{.Version}}-{{.Arch}}-{{.OS}}eabihf.{{.Format}}
```

Comment:
```console
$ cmdx lsa -repo bootandy/dust v0.8.0 | grep linux
+ REPO=${REPO#https://github.com/}
repo=$(bash scripts/get_test_pkg.sh "$REPO")

gh release view --json assets --jq ".assets[].name" -R "$repo" "$VERSION"

dust-v0.8.0-arm-unknown-linux-gnueabihf.tar.gz
dust-v0.8.0-i686-unknown-linux-gnu.tar.gz
dust-v0.8.0-i686-unknown-linux-musl.tar.gz
dust-v0.8.0-x86_64-unknown-linux-gnu.tar.gz
dust-v0.8.0-x86_64-unknown-linux-musl.tar.gz
```

I'm not familiar with Rust, but maybe `dust-v0.8.0-x86_64-unknown-linux-musl.tar.gz` or `dust-v0.8.0-x86_64-unknown-linux-gnu.tar.gz` is better.

Reply:
Ah, sorry. `x86_64` is for amd64.

Reply:
```console
$ file -s dust-v0.8.0-arm-unknown-linux-gnueabihf/dust 
dust-v0.8.0-arm-unknown-linux-gnueabihf/dust: ELF 32-bit LSB pie executable, ARM, EABI5 version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-armhf.so.3, for GNU/Linux 3.2.0, stripped
```

This binary is for 32 bit, not 64 bit.

Reply:
```console
$ cmdx con
+ bash scripts/connect.sh
[INFO] Connecting to the container aqua-registry (linux/arm64)
root@3803a6035757:/workspace# aqua which dust
/root/.local/share/aquaproj-aqua/pkgs/github_release/github.com/bootandy/dust/v0.8.0/dust-v0.8.0-arm-unknown-linux-gnueabihf.tar.gz/dust-v0.8.0-arm-unknown-linux-gnueabihf/dust
root@3803a6035757:/workspace# dust --help
FATA[0000] aqua failed                                   env=linux/arm64 error="it failed to start the process" exe_name=dust package_name=bootandy/dust package_version=v0.8.0 program=aqua program_version=2.56.0
root@3803a6035757:/workspace#
```

It doesn't work on linux/arm64.

Reply:
c3c1576c718f8de2422f53a66762680bc16ab959